content/en/docs/concepts/storage/volumes.md

```diff
@@ -834,25 +818,21 @@ before using it in the Pod.
 
 For more details, see the [Portworx volume](https://github.com/kubernetes/examples/tree/master/staging/volumes/portworx/README.md) examples.
 
+#### Portworx CSI migration
+{{< feature-state for_k8s_version="v1.25" state="beta" >}}
+
+The `CSIMigration` feature for Portworx has been added but disabled by default in Kubernetes 1.23 since it's in alpha state.
```

Comment:
Should line 824 be updated to talk about the beta (off-by-default) state in 1.25?